src/lib/controllers/custodial.rs

```diff
@@ -14,27 +13,23 @@ pub async fn index() -> Result<HttpResponse, Error> {
     Ok(HttpResponse::Ok().body("Hello world!"))
 }
 
-pub async fn heartbeat() -> Result<HttpResponse, Error> {
-    Ok(HttpResponse::Ok().body("OK"))
+// TODO wrap in a macro to prevent prod compile?
+pub async fn error_log() -> Result<HttpResponse, Error> {
```

Comment:
Sample error: https://sentry.io/organizations/mozilla/issues/3165526179

Reply:
Also, should all these be wrapped in some kind of macro that prevents them from compiling if the environment is prod? I'm not sure how you would go about this or if it's something we actually want.

Reply:
I think it's fine.